### DISCUSSION TITLE
How to make a policy that checks if a user id is in a table?

### DISCUSSION THREAD

**Discussion by ssebastianoo** [supabase/supabase#8120] [Questions] [answered]
What SQL string should I use to make a policy for `insert` that checks if `uid()` is in a table?
the table is made like this, those are user ids:
![image](https://user-images.githubusercontent.com/48064277/183224064-cd35b02b-0606-414c-b5ef-25cad37df98e.png)

**Comment by GaryAustin1** [comment, marked as answer]
Probably
```
  exists (
    select 1 from my_id_table
    where my_id_table.id = auth.uid()
    )
```

You will need both insert "with check" policy and select policy allowed for the user https://supabase.com/docs/reference/javascript/insert#notes

**Reply by ssebastianoo** [reply]
It worked, thanks a lot!
